React/CxxBridge/RCTCxxBridge.mm

```diff
@@ -1125,6 +1128,8 @@ - (void)reloadWithReason:(NSString *)reason
   RCTTriggerReloadCommandListeners(reason);
 }
 
+#pragma clang diagnostic pop
```

Comment:
Same as before, this should be tracked in some way. Perhaps the alternative to an issue is to leave comments pointing to these call-sites from the deprecated implementation?

Reply:
Since `RCTCxxBridge` inherits from `RCTBridge`, I went ahead and just removed these overrides, since `RCTBridge` now has the same implementation as this (minus the warn log).